src/exasol_odbc/connectionBuilder.js

```diff
@@ -1,22 +1,39 @@
-(function dsbuilder(attr)
-{
-    var params = {};
+(function dsbuilder(attr) {
+    "use strict";
+
+    const odbcDriverDebugEnabled = false;
+    const odbcDriverLogFile = "C:\\tmp\\exa-odbc.log"; // Windows
+    //const odbcDriverLogFile = "/tmp/exa-odbc.log"; // Linux
+
+    function log(str) {
+        logging.Log("[connectionBuilder.js] " + str)
+    }
+
+    const params = {};
+
+    if (odbcDriverDebugEnabled) {
+        log("ODBC Driver log enabled. Log file: " + odbcDriverLogFile);
+        params["EXALOGFILE"] = odbcDriverLogFile;
+        params["LOGMODE"] = "DEFAULT";
+    }
 
     params["EXAHOST"] = attr[connectionHelper.attributeServer];
     params["EXAPORT"] = attr[connectionHelper.attributePort];
 
-    if (attr["schema"] != "")
+    if (attr["schema"] && attr["schema"] != "") {
```

Comment:
Why the duplicate?

Reply:
This handles the case when `attr["schema"]` is `undefined`. Without this, the output would contain `"EXASCHEMA=undefined"`